client/src/services/selectedNode/reducer.tsx

```diff
@@ -3,7 +3,8 @@ import { ISimpleAction } from 'types';
 const initialState: any = {
   source: null,
   resource: null,
-  attribute: null
+  attribute: null,
+  selectedInputGroup: 0
```

Comment:
i don't think it's a good idea to initialize this to 0, I would use null instead. Would that cause an issue elsewhere ?